draft-ietf-ohai-ohttp.md

```diff
@@ -1001,10 +1001,21 @@ A proxy MAY add information to requests if the client is aware of the nature of
 the information that could be added.  The client does not need to be aware of
 the exact value added for each request, but needs to know the range of possible
 values the proxy might use.  It is important to note that information added by
-the proxy can reduce the size of the anonymity set of clients at a server.
-
-A proxy can also generate responses, though it assumed to not be able to
-examine the content of a request (other than to observe the choice of key
+the proxy can reduce the size of the anonymity set of clients at a target server.
+
+Proxies MAY apply differential treatment to clients that engage in abusive
+behavior, e.g., by sending too many requests in comparison to other clients,
+or as a response to rate limits signalled from the target. Any such
```

Comment:
Check terminology: is target right? I think that gateway is now correct.

Reply:
Yeah, I'll update this once we land the terminology PR.

Reply:
I just wanted to get this clear.  Because target != gateway, which is important.

In the new terms, the relay might receive feedback from the gateway about specific requests.  If it does so and changes its treatment as a result, that generates a new type of signal.  The client needs to understand the sorts of changes in treatment that might occur and accept those before using a relay.

The target might also provide feedback to the gateway.  The gateway might use that feedback, but it is only if it forwards that information to the relay that it might have an effect on privacy.

Reply:
Right -- the intent here is the gateway, not the target.